lib/bundle/brew_dumper.rb

```diff
@@ -32,7 +32,7 @@ def dump
         brewline += "brew \"#{f[:full_name]}\""
         args = f[:args].map { |arg| "\"#{arg}\"" }.sort.join(", ")
         brewline += ", args: [#{args}]" unless f[:args].empty?
-        brewline += ", restart_service: true" if BrewServices.started?(f[:full_name])
+        brewline += ", restart_service: true" if !ARGV.include?("--no-restart") && BrewServices.started?(f[:full_name])
```

Comment:
Can you adjust this to use a block form `if` (i.e. with an `end`) so the line isn't so long?